packages/core-data/src/resolvers.js

```diff
@@ -631,16 +641,9 @@ export const getUserPatternCategories =
 			}
 		);
 
-		const mappedPatternCategories =
-			patternCategories?.map( ( userCategory ) => ( {
-				...userCategory,
-				label: userCategory.name,
-				name: userCategory.slug,
-			} ) ) || [];
-
```

Comment:
I thought it was okay to manipulate data in resolvers. It shouldn't care about referencial stability before settings data in a store.

Reply:
I'm not sure TBH, but yeah I think you're right and it makes sense. Let me try reverting this part.